server/database/data/badges.json

```diff
@@ -0,0 +1,22 @@
+[
+    {
+    "name": "Maître des recettes",
+    "description": "Tu as posté plus de 10 recettes, félicitations",
+    "image": "/images/Badge_id1.png"
```

Comment:
your badge images aren't in the declared folder.

Reply:
It's done